sources/packages/web/src/components/common/notes/Notes.vue

```diff
@@ -108,9 +107,10 @@ export default {
   setup(props: any, context: any) {
```

Comment:
pls use `defineComponent` for all `setup`

Reply:
For certain pages, the properties in the component are not created type specific and it takes time to resolve it. So we will create a tech debt to convert the existing one which does not use defineComponent to use it.